nautobot_golden_config/views.py

```diff
@@ -448,9 +465,19 @@ class GoldenConfigSettingUIViewSet(views.NautobotUIViewSet):
     serializer_class = serializers.GoldenConfigSettingSerializer
     table_class = tables.GoldenConfigSettingTable
     lookup_field = "pk"
+    object_detail_content = details.golden_config_setting
+    extra_buttons = ("clone",)
+
+    def _get_dg_context(self, instance):
```

Comment:
might as well make a static method if not leveraging instance or class attributes/methods.